spec/services/metrics/project_review_turnaround_processor_spec.rb

```diff
@@ -0,0 +1,190 @@
+require_relative '../../support/metrics_specs_helper'
+
+RSpec.describe Metrics::ReviewTurnaroundPerProjectProcessor do
+  subject { Metrics::ReviewTurnaroundPerProjectProcessor }
+
+  let(:test_repository_a_payload) do
+    (build :repository_payload, name: 'Project A')['repository']
+  end
+
+  let(:create_test_events) {}
+
+  let(:time_interval_to_process) do
+    TimeInterval.new(starting_at: Time.zone.parse('2020-01-01 00:00:00'),
+                     duration: 1.day)
+  end
+
+  before do
+    create_test_events
+    process_all_events
+  end
+
+  context 'no review_turnaround events' do
+    describe 'when processing a collection containing no review_events' do
+      let(:create_test_events) do
+        pull_request_event_payload = create_pull_request_event(
+          action: 'opened',
+          created_at: Time.zone.parse('2020-01-01T15:10:00')
+        )
+
+        create_review_comment_event pull_request_event_payload: pull_request_event_payload
```

Comment:
use parenthesis in method calls, unless some specific cases but I think this is not the case. https://github.com/rootstrap/tech-guides/tree/master/ruby#method-invocation-parens

Reply:
Fixed it. Thanks!